movingpandas/trajectory_aggregator.py

```diff
@@ -229,15 +237,28 @@ def __init__(self, cells, traj_collection):
         self.cells = cells
         self.cells_union = cells.geometry.unary_union
 
-        self.id_to_centroid = {i: [f, [0, 0, 0, 0, 0]] for i, f in cells.iterrows()}
+        self.id_to_centroid = {
+            i: [
+                f,
+                [
+                    0,
+                    0,
+                    0,
+                    0,
+                    0,
+                ],
+            ]
+            for i, f in cells.iterrows()
+        }
```

Comment:
I understand that the original line can be made more readable by breaking it into 2-3 lines but these 12 lines seem excessive and counterproductive.